docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -108,7 +169,7 @@ Run the ``rs.add()`` command with the FQDN and port number of the other instance
    PRIMARY> rs.add("<fqdn>:<port>")
```

Comment:
There is no `fqdn` anymore as that's handled by the Kubernetes Service.
This should be the `mdb-instance-name` for the new node to be added.